py3status/modules/README.md

```diff
@@ -2722,6 +2724,297 @@ Requires:
+      `{temp}`, `{sunrise}`, `{sunset}` `{descript}`, `{descript_long}`,
+      `{forecast}`
+
+    You may also use the icons in the icon dictionary with their identifiers
+
+    *(default '{icon}: {temp}')*
+
+  - `format_clouds`: Formatting for cloud coverage (percentage).
+    Available placeholders:
+      `{icon}`, `{coverage}`
+
+    *(default '{icon}: {coverage}%')*
+
+  - `format_rain`: Formatting for rain volume over the past 3 hours
+    Available placeholders:
+      `{icon}`, `{mm}`, `{cm}`, `{m}`, `{in}`, `{ft}`, `{yrd}`
+
+    *(default '{icon}: {in} inches')*
+
+  - `format_snow`: Formatting for snow volume over the past 3 hours
+    Available placeholders:
+      `{icon}`, `{mm}`, `{cm}`, `{m}`, `{in}`, `{ft}`, `{yrd}`
+
+    *(default '{icon}: {in} inches')*
+
+  - `format_wind`: Formatting for wind degree and speed
+    Available placeholders:
+      `{icon}`, `{deg}`, `{msec_speed}`, `{kmh_speed}`, `{fsec_speed}`,
+      `{mph_speed}`, `{msec_gust}`, `{kmh_gust}`, `{fsec_gust}`, `{mph_gust}`
+
+    *(default '{icon}: {mph_speed} mph')*
+
+  - `format_humidity`: Formatting for humidity (percentage)
+    Available placeholders:
+      `{icon}`, `{humid}`
+
+    *(default '{icon}: {humid}%')*
+
+  - `format_pressure`: Formatting for atmospheric pressure
+    Available placeholders:
+      `{icon}`, `{press}`, `{sea_level}`
+
+    *(default '{icon}: {press} hPa')*
+
+  - `format_temp`: Formatting for temperature
+    Available placeholders:
+      `{icon}`, `{c}`, `{c_min}`, `{c_max}`, `{f}`, `{f_min}`, `{f_max}`,
+      `{k}`, `{k_min}`, `{k_max}`
+
+    *(default '{icon}: {f}°')*
+
+  - `format_sunrise_time`: Formatting string for sunrise time
+    This follows the datetime.strftime() spec.
+
+  - `format_sunrise`: Formatting for sunrise time
+    Available placeholders:
+      `{icon}`, `{strftime}`
+
+    *(default '{icon}: {strftime}')*
+
+  - `format_sunset_time`: Formatting string for sunset time
+    This follows the
+    [`datetime.strftime()`](https://docs.python.org/2/library/datetime.html#strftime-strptime-behavior)
+    spec.
+
+  - `format_sunset`: Formatting for sunset time
+    Available placeholders:
+      `{icon}`, `{strftime}`
+
+    *(default '{icon}: {strftime}')*
+
+  - `forecast_format`: Formatting for future forecasts
+    Available placeholders:
+      `{icon}`, `{clouds}`, `{snow}`, `{wind}`, `{humidity}`, `{pressure}`,
+      `{temp}`, `{descript}`, `{descript_long}`
+
+    This is similar to the 'format' field, but contains information
+    for future weather
+
+    *(default '{icon}')*
+
+  - `forecast_separator`: Separator between entries in the forecast
+    *(default ' ')*
+
+  - `forecast_today`: Include today in the forecast? (Boolean)
+    *(default False)*
+
+  - `forecast_num`: Number of days to include in the forecast, including today
+    (regardless of the 'forecast_today' flag)
+
+    *(default 0)*
+
+Format Placeholders:
```

Comment:
I think it would be nice to break up placeholders into their own `format_*` sections so if somebody want something for `format_wind` or `format_temperature`, they can glance at `format_*` placeholders list instead of this merged list... Make the information easier for others to parse. IMHO.